app/controllers/api/alert_definitions_controller.rb

```diff
@@ -1,26 +1,39 @@
 module Api
   class AlertDefinitionsController < BaseController
-    REQUIRED_FIELDS = %w(description db expression options).freeze
+    REQUIRED_FIELDS = %w(description db options).freeze
```

Comment:
I know that you're only changing this part - but I'm also confused as to why these "required fields" aren't expressed in the model too.

Reply:
Agreed, I can remove them in this PR and then add the validations in the model.